tests/comfy_cli/test_knowledge.py

```
@@ -0,0 +1,622 @@
+
+        def fake_plain(req, **kw):
+            seen["url"] = req.full_url
+            seen["ua"] = req.get_header("User-agent")
+            return self._fake_resp(b'{"models": {}}')
+
+        monkeypatch.setattr(knowledge, "plain_urlopen", fake_plain)
+        monkeypatch.setattr(knowledge, "authed_urlopen", lambda *a, **kw: pytest.fail("authed opener used"))
+        assert knowledge._http_get("https://example.com/knowledge.json") == b'{"models": {}}'
+        assert seen == {"url": "https://example.com/knowledge.json", "ua": "comfy-cli"}
+
+    def test_non_200_and_oversize_raise(self, monkeypatch):
```

Pylint (4.0.6)

[convention] Missing function or method docstring

(C0116)